source/beamline/PGM/PGMBpmControl.h

```diff
@@ -0,0 +1,55 @@
+#ifndef PGMBPMCONTROL_H
+#define PGMBPMCONTROL_H
+
+#include <QObject>
```

Comment:
QObject include is technically unnecessary because you are already inheriting it from AMPVControl.